src/debug_utils.cc

```diff
@@ -299,6 +299,106 @@ void CheckedUvLoopClose(uv_loop_t* loop) {
   CHECK(0 && "uv_loop_close() while having open handles");
 }
 
+std::vector<std::string> NativeSymbolDebuggingContext::GetLoadedLibraries() {
+  std::vector<std::string> list(0);
+#ifdef __linux__
+  dl_iterate_phdr(
+      [](struct dl_phdr_info* info, size_t size, void* data) {
+        std::vector<std::string>* list = static_cast<std::vector<std::string>*>(data);
+        if (info->dlpi_name != nullptr && *info->dlpi_name != '\0') {
+          list->push_back(info->dlpi_name);
+        }
+        return 0;
+      },
+      &list);
+#elif __APPLE__
+  int i = 0;
```

Comment:
Should be a `uint32_t`, that's what `_dyld_get_image_name()` accepts.

Reply:
used the right type